.github/workflows/do_release.yml

```diff
@@ -20,4 +20,4 @@ jobs:
           # Use the changelog re-calculated to create a new release
           gh release create ${VERSION} --draft --title ${VERSION} -n "${CHANGELOG}"
         env:
-          GH_TOKEN: ${{ secrets.RELEASEBOT_GITHUB_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
does it mean it's not working based on the problems with the roles script?

Reply:
Correct. I've updated the commit. Thanks!